packages/app-extensions/src/form/validators/type.js

```diff
@@ -18,24 +17,25 @@ export const urlValidator = value => {
 }
 
 const DEFAULT_DEFAULT_COUNTRY = 'CH'
-export const phoneValidator = (value, fieldModel = {}) => {
+export const phoneValidator = async(value, fieldModel = {}) => {
+  const libphone = await import(/* webpackChunkName: "libphonenumber-js" */ 'libphonenumber-js')
+
   const isValidCustomRegex = fieldModel.customPhoneRegex && value.match(fieldModel.customPhoneRegex)
-  const isValidPhoneNumber = isValidNumber(value, (fieldModel.defaultCountry) || DEFAULT_DEFAULT_COUNTRY)
+  const isValidPhoneNumber = libphone.isValidNumber(value, (fieldModel.defaultCountry) || DEFAULT_DEFAULT_COUNTRY)
 
-  if (isValidCustomRegex || isValidPhoneNumber) {
+  if (isValidCustomRegex || isValidPhoneNumber || !value) {
     return null
   }
 
-  return {
+  return new Promise(resolve => resolve({
```

Comment:
whats the point of this promise when you resolve it immediately?

Reply:
true